packages/logger-plugin/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "@prodo/logger",
+  "version": "0.0.1",
+  "main": "lib/index.js",
+  "license": "MIT",
+  "private": true,
```

Comment:
Not sure if you want the logger plugin published yet, but if you do this line should be removed.

Reply:
good catch!